### DISCUSSION TITLE
关于actionCdKey和actionMaximumKey的一个疑惑

### DISCUSSION THREAD

**Discussion by adproqwq** [gkd-kit/gkd#538] [General]
actionCdKey和actionMaximumKey（以下简称为action key）都只能设置单个值，假设：
现在rules里有3条规则，分别为key1、2、3，该如何设置字段使任意一个触发而其他都进入冷却呢？

我的想法是：
key1：设置action key为2，
key2：设置action key为1，
key3：设置action key为2。

这样的设置是否可以满足上述假设的要求？

**Comment by lisonge** [comment]
只需要按如下设置就行
```txt
key1: key=1
key2: actionCdKey=1
key3: actionCdKey=1
```

每个规则默认情况下有自己的计数器和计时器
actionCdKey/actionMaximumKey 的作用就是更改这个计数器和计时器的引用为其他规则的引用

**Reply by adproqwq** [reply]
了解
